### ISSUE TITLE
Please update the rules and regulations web site as follows:

### ISSUE THREAD

**Issue by luisaguilar-onrrgov** [ONRR/onrr.gov-site#316]
Hello, Please update the rules and regulations web pages as follows:

Under the following link:

https://www.onrr.gov/Laws_R_D/FRNotices/ICR0136.htm

And under:

30 CFR Parts 1202, 1204, and 1206 Federal Oil and Gas Valuation
Please change:

(Expires March 31, 2020)

to:

(Expires April 30, 2023)

Under the expiration date and above the last notice "OMB Notice of Action Dated March 2, 2017"

Please post the attached OMB Notice as follows:

OMB Notice of Action Dated April 30, 2023 and hyperlink with attachment.

[NOA_ICR 1012-0005_Exp 4.30.2023.pdf](https://github.com/ONRR/onrr.gov-site/files/5355773/NOA_ICR.1012-0005_Exp.4.30.2023.pdf)

Under Forms, please replace PDF with updated form - see attachment.

[Form ONRR-4393_Expires_4.30.2023.pdf](https://github.com/ONRR/onrr.gov-site/files/5355782/Form.ONRR-4393_Expires_4.30.2023.pdf)

Under "Federal Register Publications and Comments" and above the last posting

Please add the following:

30-Day Agency Information Collection (85 FR 5033--January 28, 2020)

https://www.federalregister.gov/documents/2020/01/28/2020-01360/agency-information-collection-activities-submission-to-the-office-of-management-and-budget-for

Please let us know if you have any questions.

**Comment by lpgoldstein** [comment]
@luisaguilar-onrrgov I've attached the pdfs that have the accessibility reports attached. I've also attached the separate accessibility reports for two documents. I'm unable to do it for the 30-day notice document for some reason, but the pdf I've attached has the accessibility report in it. I've followed up by email as well. 
[30-Day.Notice_accessibility report attached.pdf](https://github.com/ONRR/onrr.gov-site/files/5356194/30-Day.Notice_accessibility.report.attached.pdf)

[Form.ONRR-4393_Expires_4.30.2023.pdf](https://github.com/ONRR/onrr.gov-site/files/5356190/Form.ONRR-4393_Expires_4.30.2023.pdf)
[Form.ONRR-4393_Expires_4.30.2023.pdf.docx](https://github.com/ONRR/onrr.gov-site/files/5356191/Form.ONRR-4393_Expires_4.30.2023.pdf.docx)
[NOA_ICR.1012-0005_Exp.4.30.2023.pdf](https://github.com/ONRR/onrr.gov-site/files/5356192/NOA_ICR.1012-0005_Exp.4.30.2023.pdf)
[NOA_ICR.1012-0005_Exp.4.30.2023.pdf.docx](https://github.com/ONRR/onrr.gov-site/files/5356193/NOA_ICR.1012-0005_Exp.4.30.2023.pdf.docx)

**Comment by lpgoldstein** [comment]
@luisaguilar-onrrgov if you update this request with the proper links I can go ahead and take care of these updates today.

**Comment by lpgoldstein** [comment]
@luisaguilar-onrrgov I'm going to go ahead and close this issue since there haven't been any updates. Please create a new issue when you want us to move forward with this.
